### ISSUE TITLE
Error 404 in Bio.ExPASy 

### ISSUE THREAD

**Issue by jlalmeidaf** [biopython/biopython#1145]
Good morning,

I get error 404 when i use Bio.ExPASy.get_prosite_raw(), Bio.ExPASy.get_prosite_entry() and Bio.ExPASY.get_prodoc_entry().
Thanks

**Comment by peterjc** [comment]
Which version of Biopython do you have? You can check with:

```python
>>> import Bio
>>> print(Bio.__version)
...
```

**Comment by peterjc** [comment]
It looks like only ``Bio.ExPASy.get_sprot_raw(...)`` has unit test coverage, which explains why we didn't notice till your report:

https://codecov.io/gh/biopython/biopython/src/master/Bio/ExPASy/__init__.py

Could you give a specific example in each case which ought to work? i.e. Sensible ID for a small record which we could use for a new test.

**Comment by jlalmeidaf** [comment]
My version is 1.67

I really do not know what the output of these methods would be like, I'm just reading the documentation for this site http://biopython.org/DIST/docs/tutorial/Tutorial.html#htoc144. In session 10.5.3 you have the use of these methods. I believe the method "handle = ExPASy.get_prosite_raw ('PDOC00001')" returns a  Bio.Prosite.Record object

**Comment by peterjc** [comment]
Ah - that helps, I've never used these functions myself but should have checked there. e.g.

```
>>> from Bio import ExPASy
>>> from Bio.ExPASy import Prodoc
>>> handle = ExPASy.get_prosite_raw('PDOC00001')
>>> record = Prodoc.read(handle)
```

I'm pretty sure the ExPASy site will explain the new URL structure somewhere - or anyone familiar with the site can probably figure it out...

Cross referencing #1072 which explores including the online doctests into ``test_Tutorial.py``.

**Comment by peterjc** [comment]
This looks like the raw data URL we need: http://prosite.expasy.org/PDOC00001.txt

**Comment by peterjc** [comment]
It looks like for Python 3 we might want to return a text mode handle (as expected in our parser), as done in e.g. ``Bio.TogoWS`` and ``Bio.Entrez``...

**Comment by peterjc** [comment]
The Prosite parser will need updating too:

```
>>> from Bio.ExPASy import Prosite
>>> with ExPASy.get_prosite_raw('PS00001') as handle: 
...     record = Prosite.read(handle) 
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "/Users/pc40583/repositories/biopython/Bio/ExPASy/Prosite.py", line 48, in read
    record = __read(handle)
  File "/Users/pc40583/repositories/biopython/Bio/ExPASy/Prosite.py", line 172, in __read
    raise ValueError("I don't understand date line\n%s" % line)
ValueError: I don't understand date line
DT   01-APR-1990 CREATED; 01-APR-1990 DATA UPDATE; 01-APR-1990 INFO UPDATE.
```

**Comment by peterjc** [comment]
Proposed fix on pull request #1215
